docs/src/network.md

```diff
@@ -2,10 +2,16 @@
 id: network
 title: "Network"
 ---
+## Introduction
 
-Playwright provides APIs to **monitor** and **modify** network traffic, both HTTP and HTTPS. Any requests that a page does, including [XHRs](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) and
+Playwright provides APIs to **monitor** and **modify** browser network traffic, both HTTP and HTTPS. Any requests that a page does, including [XHRs](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) and
 [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) requests, can be tracked, modified and handled.
 
+Check out our [API mocking guide](./mock.md) to learn more on how to 
+- mock API requests and never hit the API
+- perform the API request and fulfill it with a modified response
+- use HAR files to mock network requests.
+
 ## Network mocking
```

Comment:
Drop this section now that we redirect to the mocking guide? Same goes for "Handle requests" and "Modify responses" sections. We should also move "Abort requests" and "Modify requests" to the mocking guide - can be done in another PR.